app/forms/validators.py

```diff
@@ -128,39 +128,64 @@ def __call__(
         field: Union[DecimalFieldWithSeparator, IntegerFieldWithSeparator],
     ) -> None:
         value: Union[int, Decimal] = field.data
+        decimal_places = 0
```

Comment:
We should set this to None, as the `custom_format_currency` will set to 0.
How come it is not needed for `format_number`?.

Reply:
Set to None now 👍